### PR TITLE
Integrate service zapier

### PR DESCRIPTION
**Pull request by blackinno**
- Update zapier service to eventsList.
- Update zapier templates.

Relate to https://github.com/mesg-foundation/aragon/pull/46

### CODE REVIEW

File: app/src/utils/eventsList.js [hunk 1]
```diff
@@ -74,6 +59,13 @@ export default {
         { name: 'subject', label: 'Subject', type: 'text', required: true },
         { name: 'text', label: 'Content', type: 'text', required: false }
       ]
+    },
+    {
+      name: 'Zapier',
+      label: 'Send data to Zap',
```
Comment (NicolasMahe) [inline app/src/utils/eventsList.js:65]:
```suggestion
      label: 'Trigger a Zap',
```

File: app/src/utils/eventsList.js [hunk 2]
```diff
@@ -74,6 +59,13 @@ export default {
...
+      instanceHash: process.env.STORE_HASH,
+      taskKey: 'execute',
```
Comment (NicolasMahe) [inline app/src/utils/eventsList.js:67]: what is this taskKey referring to?

File: app/src/utils/eventsList.js [hunk 3]
```diff
@@ -74,6 +59,13 @@ export default {
...
+      inputsFields: [{ name: 'zapId', label: 'Zap ID', type: 'text', placeholder: '90449527', required: true }]
```
Comment (NicolasMahe) [inline app/src/utils/eventsList.js:68]: can you add more explanation on how to get the zap id.
Something like:
> You can find your zap id in the URL of the zap after creating it. Eg: https://zapier/ZAP_ID/XXXX

File: app/src/utils/eventsList.js (hunk 1 above)
Comment (blackinno) [reply app/src/utils/eventsList.js]: Fixed.

File: app/src/utils/eventsList.js (hunk 3 above)
Comment (blackinno) [reply app/src/utils/eventsList.js:68]: Done.

File: app/src/utils/eventsList.js (hunk 2 above)
Comment (blackinno) [reply app/src/utils/eventsList.js]: This taskKeys isn't referring to any I forget to remove it. Now it's removed.

**Pull request merged**